### PR TITLE
updated health probes

### PR DESCRIPTION
**Pull request by xrajesh**
**Related Issue:**  open-cluster-management/backlog#<ISSUE_NUMBER>

### Description of changes

/hold
`/hold`

### CODE REVIEW

**Review by zlayne — APPROVED** [review]
lgtm

**Comment by jlpadilla** [comment]
Can we add a test? Everything else looks good.

/lgtm

**Review by jlpadilla — APPROVED** [review]

File: pkg/handlers/healthProbes.go [hunk 1]
```diff
@@ -7,17 +7,39 @@ import (
...
-	glog.V(2).Info("livenessProbe")
+	glog.V(2).Info("livenessProbe - Checking local cluster id.")
+	monitor := monitor.NewClusterMonitor()
+
+	//Get local-cluster id , if -1 is returned then service will
+	// not be able to get to cloud.redhat.com
+	if monitor.GetLocalCluster() == "-1" {
+		// Respond with error.
+		glog.Warning("Cannot get local-cluster id.")
+		http.Error(w, "Cannot get local-cluster id.", 503)
```
Comment (SherinV) [inline pkg/handlers/healthProbes.go:24]: Maybe, add `Cannot reach CRC` to the error message to make it clearer what the error means.

**Review by SherinV — APPROVED** [review]

**Comment by xrajesh** [comment]
`/unhold`

**Pull request merged**